<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Phraktalus</class>
 <widget class="QMainWindow" name="Phraktalus">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Phraktalus</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/phraktalus.png</normaloff>:/icons/phraktalus.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Process">
    <property name="title">
     <string>&amp;Process</string>
    </property>
    <addaction name="action_Previous"/>
    <addaction name="action_Next"/>
    <addaction name="action_Zoom"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menu_Grids">
     <property name="title">
      <string>&amp;Grids</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/icons/rule.png</normaloff>:/icons/rule.png</iconset>
     </property>
     <addaction name="action_Vertical"/>
     <addaction name="action_Horizontal"/>
    </widget>
    <addaction name="action_Toolbar"/>
    <addaction name="separator"/>
    <addaction name="menu_Grids"/>
    <addaction name="action_Show_Labels"/>
    <addaction name="action_Surface"/>
    <addaction name="action_Lighting"/>
    <addaction name="action_Reset_Camera"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Process"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="action_Open">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/open.png</normaloff>:/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="toolTip">
    <string>Opens a file containing digital elavation data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/close.png</normaloff>:/icons/close.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Reset_Camera">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/camera.png</normaloff>:/icons/camera.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Reset Camera</string>
   </property>
   <property name="toolTip">
    <string>Resets the camera to the initial state</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="action_Toolbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Toolbar</string>
   </property>
   <property name="toolTip">
    <string>Show/Hide the toolbar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About Phraktalus</string>
   </property>
   <property name="toolTip">
    <string>About Phraktalus</string>
   </property>
  </action>
  <action name="action_Vertical">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show &amp;Vertical Grid</string>
   </property>
   <property name="toolTip">
    <string>Toggles vertical grid</string>
   </property>
  </action>
  <action name="action_Horizontal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show &amp;Horizontal Grid</string>
   </property>
   <property name="toolTip">
    <string>Toggles horizontal grid</string>
   </property>
  </action>
  <action name="action_Surface">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/brush.png</normaloff>:/icons/brush.png</iconset>
   </property>
   <property name="text">
    <string>Show &amp;Surface</string>
   </property>
   <property name="toolTip">
    <string>Toggles surface's filling</string>
   </property>
  </action>
  <action name="action_Zoom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/zoom.png</normaloff>:/icons/zoom.png</iconset>
   </property>
   <property name="text">
    <string>Zoom</string>
   </property>
   <property name="toolTip">
    <string>Select height map subregion to view</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_Show_Labels">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/labels.png</normaloff>:/icons/labels.png</iconset>
   </property>
   <property name="text">
    <string>Show Labels</string>
   </property>
   <property name="toolTip">
    <string>Shows/Hides the labels in the viewport</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="action_New">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/new.png</normaloff>:/icons/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>Creates a new terrain using the built-in algorithms</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Previous">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/previous.png</normaloff>:/icons/previous.png</iconset>
   </property>
   <property name="text">
    <string>Previous</string>
   </property>
   <property name="toolTip">
    <string>Go back to the previous model</string>
   </property>
   <property name="shortcut">
    <string>Backspace</string>
   </property>
  </action>
  <action name="action_Next">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/next.png</normaloff>:/icons/next.png</iconset>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
   <property name="toolTip">
    <string>Shows the next zoom level</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </action>
  <action name="action_Lighting">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/sun.png</normaloff>:/icons/sun.png</iconset>
   </property>
   <property name="text">
    <string>Lighting</string>
   </property>
   <property name="toolTip">
    <string>Toggles lighting on and off in the 3D scene renderer</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>action_Quit</sender>
   <signal>triggered()</signal>
   <receiver>Phraktalus</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Toolbar</sender>
   <signal>toggled(bool)</signal>
   <receiver>toolBar</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
